src/dialogs/voice-command-dialog/ha-voice-command-dialog.ts

```diff
@@ -312,6 +319,9 @@ export class HaVoiceCommandDialog extends LitElement {
           margin: 0 24px 16px;
           min-height: 399px;
         }
+        ha-tip {
+          padding-bottom: 14px;
```

Comment:
```suggestion
          padding-bottom: 16px;
```

We try to use a 8px grid

Reply:
Ah good to know 🙂